swabbie-core/src/main/kotlin/com/netflix/spinnaker/swabbie/AbstractResourceTypeHandler.kt

```diff
@@ -440,6 +442,11 @@ abstract class AbstractResourceTypeHandler<T : Resource>(
     action: Action
   ) {
     val totalProcessed = Math.min(getMaxItemsProcessedPerCycle(workConfiguration), totalResourcesVisitedCounter.get())
+    registry.counter(numberOfCandidates.withTags(
+      "actionName", action.name,
+      "resourceType", workConfiguration.resourceType,
+      "candidates", candidateCounter.get().toString()
+    )).increment()
```

Comment:
I don't think this is the correct way to do this. This will publish a metric with a tag of the value, which is not what we want. I think we want to use a [PolledMeter](https://github.com/Netflix/spectator/blob/master/spectator-api/src/main/java/com/netflix/spectator/api/patterns/PolledMeter.java#L79)

Reply:
(or more realistically a gauge)

Reply:
+1 a gauge is more appropriate than a simple counter here.